lib/seahorn/BvOpSem2.cc

```diff
@@ -3182,6 +3186,45 @@ Optional<APInt> Bv2OpSem::agg(Type *aggTy,
   return res;
 }
 
+/// Crab stuff
+// Build Crab CFG, run crab analyses, etc.
+void Bv2OpSem::mkCrabAnalysis(const llvm::Module &M,
+                              seadsa::GlobalAnalysis &dsa,
+                              TargetLibraryInfoWrapperPass &tli) {
+
+  std::unique_ptr<clam::HeapAbstraction> heap_abs(
+      new clam::SeaDsaHeapAbstraction(M, dsa));
+
+  // -- Set parameters for CFG
+  clam::CrabBuilderParams cfg_builder_params;
+  LOG("bmc-crab", cfg_builder_params.print_cfg = true;);
+  // Note: default precision is NUM
```

Comment:
NUM only models integer registers. We want also to reason about memory contents.
Add this line `cfg_builder_params.setPrecision(clam::CrabBuilderPrecision::MEM);`